File: Anole/HTMLBRElement.vb
Namespace Anole

    Public Class HTMLBRElement : Inherits HTMLElement

        Public Sub New(ByRef document As HTMLDocument, ByVal parent As HTMLElement, ByVal xnode As XmlNode)
            Initialize(document, parent, xnode)
        End Sub

        Public Overloads Overrides Sub Layout(ByVal renderer As HTMLRenderer, ByVal g As Graphics)
            renderer.NewBand()
            renderer.NewBand()
        End Sub
    End Class

End Namespace

File: Anole/HTMLBRElement.cs
using System;
using System.Drawing;
using System.Xml;

namespace Anole
{
	/// <summary>
	/// Summary description for HTMLBRElement.
	/// </summary>
	public class HTMLBRElement : HTMLElement
	{
		public HTMLBRElement(ref HTMLDocument document, HTMLElement parent, XmlNode xnode)
		{
			Initialize(ref document, parent, xnode);
		}

		public override void Layout(HTMLRenderer renderer, Graphics g)
		{
			renderer.NewBand();
			renderer.NewBand();
		}
	}
}
